public/locale/en.json

```diff
@@ -1715,6 +1716,7 @@
   "questionnaire_not_exist": "The questionnaire you tried to access does not exist.",
   "questionnaire_submission_failed": "Failed to submit questionnaire",
   "questionnaire_submitted_successfully": "Questionnaire submitted successfully",
+  "questionnaires": "Questionnaires",
```

Comment:
https://www.i18next.com/translation-function/plurals

Make sure to modify existing instances (including any relevant language files).